app/controllers/api/v8/submissions_controller.rb

```diff
@@ -224,7 +224,7 @@ def all_submissions
 
     submissions = authorized_content(Submission.where(course_id: course.id))
 
-    respond_with submissions
+    render json: submissions
```

Comment:
I would prefer the `present` https://github.com/testmycode/tmc-server/blob/master/app/controllers/api/beta/base_controller.rb#L21-L27 so that we could easily support e.g `?pretty=1` for easier human consumption of the json api